app/containers/Forms/FuelFields.tsx

```diff
@@ -26,22 +27,17 @@ const FuelFields: React.FunctionComponent<Props> = ({
       <Col xs={12} md={6}>
         <Form.Group controlId="id.fuelType">
           <Form.Label>Fuel Type</Form.Label>
-          <Form.Control
-            as="select"
-            value={formData.fuelType}
-            onChange={e => {
+          <SearchDropdown
+            placeholder="Select fuel or type to filter..."
+            options={query.allFuels.edges.map(({node}) => node.name)}
```

Comment:
Would be really nice if we could pass [a list of object options](https://github.com/ericgio/react-bootstrap-typeahead/blob/master/docs/Data.md#arrayobject-wdefault-labelkey) to set the `fuelId` in case there are ever multiple nodes with the same name.

Reply:
You mean options would contain items for list + id and potentially anything else we need to pass?

Reply:
Yes, instead of a list of strings we'd pass a list of objects with both `id` and `name` attributes, that way selecting the `name` in the UI would update the `id` on the form (would need `labelKey="name"` prop sent to typeahead per docs).

Reply:
Updated in last commit.

Reply:
I see. OK.

Reply:
Updated and added more props, i.e. `inputProps` and `defaultInputValue`.